examples/sign-leaf-with-ca.rs

```diff
@@ -0,0 +1,99 @@
+use rcgen::{
+	date_time_ymd, BasicConstraints, Certificate, CertificateParams, CertificateSigningRequest,
+	DnType, ExtendedKeyUsagePurpose, IsCa, KeyUsagePurpose,
+};
+use time::{Duration, OffsetDateTime};
+use x509_parser::certification_request::X509CertificationRequest;
+use x509_parser::prelude::FromDer;
+
+/// Example demonstrating signing end-endity certificate with ca
+fn main() {
+	let ca = Ca::new();
+	let entity = Entity::new();
+
+	println!("directly signed end-entity certificate:");
+	let direct = entity
+		.certificate
+		.serialize_pem_with_signer(&ca.certificate)
+		.unwrap();
+	println!("{direct}");
```

Comment:
I think it would be cleaner to have only one `println!` by re-arranging the order of things slightly. I also think `end_entity_pem` as a var name is probably easier to understand than `direct`.

```suggestion
	let end_entity_pem = entity
		.certificate
		.serialize_pem_with_signer(&ca.certificate)
		.unwrap();
	println!("directly signed end-entity certificate: {end_entity_pem}");
```